src/utils/compositions/ambassador/ambassador.module.scss

```diff
@@ -6,7 +6,7 @@
     align-items: center;
     justify-content: center;
 
-    padding: 20px 15px;
+    padding: 7.7%;
```

Comment:
the ambassador module does not get scaled using scale() anymore. to make the padding scale it must be relative or use the base size.  because the ambassador composition is being used for both overlay and panel, I do not use the base size but a relative size.